ExtCore/Control.Indexed.Compability.fs

```diff
@@ -0,0 +1,19 @@
+//
+namespace ExtCore.Control.Indexed.Compability
+
+open ExtCore
+open ExtCore.Control.Indexed
+/// Indexed-state workflows.
+[<RequireQualifiedAccess; CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
```

Comment:
`CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)` is not needed in most cases in F# 4.1, maybe remove all of them while you are at it?

Reply:
https://fslang.uservoice.com/forums/245727-f-language/suggestions/14533251-make-modulesuffix-the-default-if-a-type-has-the

Reply:
So yes, should be removed